.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       - run:
           name: Install React Native CLI and Ignite CLI
           command: |
-            sudo npm i -g ignite-cli@next react-native-cli
+            sudo npm i -g ignite-cli@latest react-native-cli
```

Comment:
FYI, @leonskim , you don't have to use the `chore: xxx` syntax here, because I'll be squashing the commits when I merge anyway. 🐐